src/unitxt/processors.py

```diff
@@ -245,3 +245,11 @@ def process_value(self, text: Any) -> Any:
         if text is None or text == "":
             return text
         return ast.literal_eval(text.strip())
+
+
+class ExtractSafeUnsafeJudgment(FieldOperator):
```

Comment:
When we have time we need to think of a generic processor for classification @yoavkatz